erpnext/assets/doctype/asset/depreciation.py

```diff
@@ -9,7 +9,7 @@
 
 def post_depreciation_entries(date=None):
 	# Return if automatic booking of asset depreciation is disabled
-	if not cint(frappe.db.get_value("Accounts Settings", None, "book_asset_depreciation_entry_automatically")):
```

Comment:
Use frappe.db.get_single_value to fetch `book_asset_depreciation_entry_automatically` properly. This is not the correct solution.  Please go through this doc for more info https://frappe.io/docs/user/en/api/database. Closing this for now